<?xml version="1.0"?>
<launch>

    <arg name="__ns" default="/my_gen3"/>
    <node pkg="kinova_arm_apps" type="kinova_moveit_client.py" name="kinova_moveit_client" output="screen">
        <param name="__ns" value="$(arg __ns)"/>
    </node>

</launch>
